indico/modules/events/papers/templates/emails/added_to_reviewing_team.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'events/papers/emails/base.html' %}
+
+{% block subject_message %}Added in paper reviewing team{% endblock %}
+
+{% block header_subtitle %}Added in paper reviewing team{% endblock %}
+
+{% block content %}
+    <p>Dear {{ receiver.first_name }},</p>
+    <p>You have been chosen as a <strong>{{ role.title }}</strong> for the conference <strong>{{ event.title }}</strong> in order to help with the paper reviewing process.</p>
+    {% if role.name == 'judge' %}
+        <p>Once there are some papers for you to judge, you will be able to do so in the judging area page:</p>
+        <p>
+            <a href="{{ url_for('papers.papers_list', event, _external=true) }}">
+                {{- url_for('papers.papers_list', event, _external=true) -}}
```

Comment:
You need to add an explicit `management=false` here and in the other emails to make sure these emails always point to the page in the display area and not the equivalent in the management area.